pfcli/group.py

```diff
@@ -187,3 +188,21 @@ def _get_current_org() -> dict:
 
     # org context may be wrong
     secho_error_and_exit("Failed to identify organization... Please set organization again.")
+
+
+@app.command(help="list up members in the current working organization")
+def members():
+    project_client: ProjectClientService = build_client(ServiceType.PROJECT)
+    group_client: GroupClientService = build_client(ServiceType.GROUP)
+    project_id = get_current_project_id()
+
+    if project_id is not None:
+        org_id = project_client.get_project(pf_project_id=project_id)["pf_group_id"]
+    else:
+        org_id = get_current_group_id()
```

Comment:
In general, we can use just `get_current_group_id`, because we should enforce the current project should belong to the current org.
I think line 199-200 is needed when we consider a malicious user who changes `$HOME/.periflow/project` file directly.